test/adagrams.test.js

```diff
@@ -144,8 +144,8 @@ describe("Adagrams", () => {
     it("accurately finds best scoring word even if not sorted", () => {
       const words = ["XXX", "XXXX", "X", "XX"];
       const correct = { word: "XXXX", score: scoreWord("XXXX") };
-
-      throw "Complete test by adding an assertion";
+      expect(correct.score).toEqual(32);
```

Comment:
It would be good to check that the correct word was returned as well. It would make sense to follow what the other tests do and do `expect(highestScoreFrom(words)).toEqual(correct);`